images/linux/scripts/installers/git.sh

```diff
@@ -46,7 +48,13 @@ DocumentInstalledItem "Git Large File Storage (LFS) ($(git-lfs --version 2>&1 |
 DocumentInstalledItem "Git-ftp ($(git-ftp --version | cut -d ' ' -f 3))"
 
 #Install hub
-snap install hub --classic
+tmp_hub="/tmp/hub"
+mkdir -p "$tmp_hub"
+url=$(curl -s https://api.github.com/repos/github/hub/releases/latest | jq -r '.assets[].browser_download_url | select(contains("hub-linux-amd64"))')
+download_with_retries "$url" "$tmp_hub"
+tar xzvf /tmp/hub-linux-amd64-*.tgz --strip-components 1 -C "$tmp_hub"
```

Comment:
It should be `tar xzvf $tmp_hub/hub-linux-amd64-*.tgz --strip-components 1 -C "$tmp_hub"`

Reply:
nice catch! fixed